### PR TITLE
Fix TestRedshiftConnectorTest TestingConnectorBehavior

### PR DESCRIPTION
**Pull request by grantatspothero**
## Description
This test does not run in CI and became stale after changing the default of testing behaviors to true: https://github.com/trinodb/trino/commit/6781fa352f959b57e457249761a48d084ac6d991

I ran tests locally but I'm not sure how to test this given this test does not run in CI. 

## Additional context and related issues
See commit here: https://github.com/trinodb/trino/commit/6781fa352f959b57e457249761a48d084ac6d991

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftConnectorTest.java [hunk 1]
```diff
@@ -82,10 +86,10 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS:
                 return false;
 
-            case SUPPORTS_AGGREGATION_PUSHDOWN_STDDEV:
-            case SUPPORTS_AGGREGATION_PUSHDOWN_VARIANCE:
-            case SUPPORTS_AGGREGATION_PUSHDOWN_COUNT_DISTINCT:
-                return true;
+            case SUPPORTS_AGGREGATION_PUSHDOWN_COVARIANCE:
```
Comment (findinpath) [inline plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftConnectorTest.java:89]: I've tested the method `io.trino.plugin.jdbc.BaseJdbcConnectorTest#testStddevAggregationPushdown` and it fails.

Comment (findinpath) [inline plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftConnectorTest.java:89]: affected tests in BCT:
- `testCovarianceAggregationPushdown`
- `testCorrAggregationPushdown`
- `testRegrAggregationPushdown`

**Comment by findinpath** [comment]
@grantatspothero I did run the tests also locally and can confirm as well that we need to set the support for the behavior to false.

```
            case SUPPORTS_AGGREGATION_PUSHDOWN_COVARIANCE:
            case SUPPORTS_AGGREGATION_PUSHDOWN_CORRELATION:
            case SUPPORTS_AGGREGATION_PUSHDOWN_REGRESSION:
                return false;
```

> I ran tests locally but I'm not sure how to test this given this test does not run in CI.


We can temporarily add a `!fixup` commit (to be dropped before merging) to include `TestRedshiftConnectorTest` to be tested as well - make sure to increase `timeout-minutes` to a higher value (e.g. : `180`) to make sure that the build doesn't time out. Afterwards we can run the PR with secrets to make sure there are no other hidden regressions.

**Comment by hashhar** [comment]
why not add `cloud-tests` profile in ci.yml? @ebyhr I see you authored https://github.com/trinodb/trino/pull/16484

**Comment by ebyhr** [comment]
ci.yml already contains `cloud-tests` profile, but the profile doesn't run `TestRedshiftConnectorTest`.

**Comment by ebyhr** [comment]
I sent #17381 within the repository. No need to push the fixup commit.

**Comment by ebyhr** [comment]
@grantatspothero @findinpath Did you run all tests of `TestRedshiftConnectorTest` locally? `testAddNotNullColumnToEmptyTable` & `testDecimalAvgPushdownForMaximumDecimalScale` failed in https://github.com/trinodb/trino/actions/runs/4912391199/jobs/8772192021?pr=17381

**Comment by grantatspothero** [comment]
Will rerun all tests on latest HEAD, I was running on a slightly old trino version and confirm.

**Comment by ebyhr** [comment]
Superseded by #17381. Thanks for preparing the base change.

**Pull request closed without merging**